it/utils/ETMPWiremock.scala

```diff
@@ -447,15 +447,6 @@ trait ETMPWiremock {
     ))
```

Comment:
There's a lot of changes needed in this file:
Anything that references ETMPPayload can be removed
There's still lots of references to v3 which need to be removed